.github/dependabot.yml

```diff
@@ -0,0 +1,257 @@
+    commit-message:
+      prefix: "[camera]"
+    schedule:
+      interval: "daily"
+    open-pull-requests-limit: 10
+
+  - package-ecosystem: "gradle"
+    directory: "/packages/espresso/example/android/app"
+    commit-message:
+      prefix: "[espresso]"
+    schedule:
+      interval: "daily"
+    open-pull-requests-limit: 10
+
+  - package-ecosystem: "gradle"
+    directory: "/packages/flutter_plugin_android_lifecycle/android"
+    commit-message:
+      prefix: "[flutter_plugin_android_lifecycle]"
+    schedule:
+      interval: "daily"
+    open-pull-requests-limit: 10
+
+  - package-ecosystem: "gradle"
+    directory: "/packages/flutter_plugin_android_lifecycle/example/android/app"
+    commit-message:
+      prefix: "[flutter_plugin_android_lifecycle]"
+    schedule:
+      interval: "daily"
+    open-pull-requests-limit: 10
+
+  - package-ecosystem: "gradle"
+    directory: "/packages/google_maps_flutter/google_maps_flutter/android"
+    commit-message:
+      prefix: "[google_maps_flutter]"
+    schedule:
+      interval: "daily"
+    open-pull-requests-limit: 10
+
+  - package-ecosystem: "gradle"
+    directory: "/packages/google_maps_flutter/google_maps_flutter/example/android/app"
+    commit-message:
+      prefix: "[google_maps_flutter]"
+    schedule:
+      interval: "daily"
+    open-pull-requests-limit: 10
+
+  - package-ecosystem: "gradle"
+    directory: "/packages/google_sign_in/google_sign_in/example/android/app"
+    commit-message:
+      prefix: "[google_sign_in]"
+    schedule:
+      interval: "daily"
+    open-pull-requests-limit: 10
+
+  - package-ecosystem: "gradle"
+    directory: "/packages/google_sign_in/google_sign_in_android/android"
+    commit-message:
+      prefix: "[google_sign_in_android]"
+    schedule:
+      interval: "daily"
+    open-pull-requests-limit: 10
+
+  - package-ecosystem: "gradle"
+    directory: "/packages/google_sign_in/google_sign_in_android/example/android/app"
+    commit-message:
+      prefix: "[google_sign_in_android]"
+    schedule:
+      interval: "daily"
+    open-pull-requests-limit: 10
+
+  - package-ecosystem: "gradle"
+    directory: "/packages/in_app_purchase/in_app_purchase_android/android"
+    commit-message:
+      prefix: "[in_app_purchase_android]"
+    schedule:
+      interval: "daily"
+    open-pull-requests-limit: 10
+
+  - package-ecosystem: "gradle"
+    directory: "/packages/in_app_purchase/in_app_purchase_android/example/android/app"
+    commit-message:
+      prefix: "[in_app_purchase_android]"
+    schedule:
+      interval: "daily"
+    open-pull-requests-limit: 10
+
+  - package-ecosystem: "gradle"
+    directory: "/packages/in_app_purchase/in_app_purchase/example/android/app"
+    commit-message:
+      prefix: "[in_app_purchase]"
+    schedule:
+      interval: "daily"
+    open-pull-requests-limit: 10
+
+  - package-ecosystem: "gradle"
+    directory: "/packages/image_picker/image_picker/example/android/app"
```

Comment:
This is missing both parts of the implementation package.

Reply:
done